dessia_common/workflow/core.py

```diff
@@ -1941,6 +1943,138 @@ def _method_jsonschemas(self):
         warnings.warn("method_jsonschema method is deprecated. Use method_schema instead", DeprecationWarning)
         return self.method_schemas
 
+    @staticmethod
+    def instantiate_objects(objects):
+        """ Instantiates objects from a list of objects. """
+        class_name = f"{objects[0].__class__.__name__}"
+        if len(objects) > 1:
+            instances = '['
+            for _ in objects:
+                inst = f"{class_name}('Put here your object')"
+                instances += inst + ", "
+            instances += ']'
+            return instances
+
+        return f"{class_name}('Put here your object')"
+
+    @staticmethod
+    def get_files(type_files):
+        """ Generates file instances from a list of file types. """
+        file_class = type_files[0].__class__.__name__
+        if len(type_files) > 1:
+            instances = "["
+            for _ in range(len(type_files)):
+                instances += f"{file_class}.from_file('Insert your path here')" + ", "
+            instances += ']'
+            return instances
+        return f"{file_class}.from_file('Insert your path here')"
+
+    @staticmethod
+    def is_object_sequence(seq):
+        """ Checks if a sequence contains serializable objects. """
+        return any(isinstance(element, SerializableObject) for element in seq)
```

Comment:
I have a feeling that this should be an "all" statement instead.
What happens in other cases ? (Not all or not any)

Reply:
in this case we consider that the sequence only contains SerializableObjects, I don't see why in the sequence we will have a SerializableObject and a file for example!

Reply:
That is why I think it is better to use an "all" statement. If you know that your sequence should be homogeneous, then you should also be conservative in your check and check for all instead of any

Reply:
done !